aea/cli/add.py

```diff
@@ -87,6 +77,40 @@ def connection(click_context, connection_name):
         logger.error(str(e))
         sys.exit(1)
 
+
+@add.command()
+@click.argument('connection_name', type=str, required=False)
```

Comment:
@panasevychol can we remove the `--id` arg and instead assume that it is passed as `connection_name`. So if the registry flag is used the `connection_name` must be a public identifier (otherwise abort) and otherwise it can be as is. This simplifies the cli and avoids us having to change the api in the future.

Reply:
The idea of having it separately was for understanding that those 2 params are completely different. I can unify that to one but that would require a bit of new logic in function. Cause it uses name in both cases (for existing connection check).

Reply:
Well, eventually we will just have the public identifier. So it makes sense to merge them now. The logic will be minimal ..if registry, apply the string split, otherwise use what is given..